accessibility-checker-engine/help/Rpt_Aria_MultipleSearchLandmarks.mdx

```diff
@@ -44,9 +44,9 @@ For example:
 
 ### About this requirement
 
-[IBM 2.4.1 Bypass Blocks](https://www.ibm.com/able/requirements/requirements/#2_4_1)
-[W3C WCAG 2.1 technique ARIA6](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA6)
-[W3C WCAG 2.1 technique ARIA13](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA13)
+* [IBM 2.4.1 Bypass Blocks](https://www.ibm.com/able/requirements/requirements/#2_4_1)
+* [WCAG technique ARIA6](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA6)
+* [WCAG technique ARIA16](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA13)
```

Comment:
Was ARIA13 and linked to 13. Now it says ARIA16 and still links to 13?

Reply:
shoud say ARIA13 and lik to ARIA13, good catch.  I changed it and pushed.